### ISSUE TITLE
Add search for public uploaded packs

### ISSUE THREAD

**Issue by cmbirk** [GovReady/Issue-Packs-Site#25]
There's a laravel trait that can be used here.  This is dependent upon #24

**Comment by cmbirk** [comment]
This is related to https://github.com/nicolaslopezj/searchable/issues/112

**Comment by gregelin** [comment]
This can be lower priority until we have more items to search. Definitely not needed for June, probably not even for July.
